### PR TITLE
[ASAP-50] delete entries from algolia when deleted from CMS

### PR DESCRIPTION
**Pull request by marianasrv**
AWS link for outputs logs: https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/functions/gp2-hub-3891-algoliaIndexOutput?tab=monitoring
Algolia index: https://dashboard.algolia.com/apps/R44097HEU2/explorer/browse/gp2-hub_CI-3891?searchMode=search
CMS env: https://app.contentful.com/spaces/6ekgyp1432o9/environments/gp2-3891/entries?id=YkBELf8xah5KqpIY&title=Overview&contentTypeId=outputs&order.fieldId=updatedAt&order.direction=descending&displayedFieldIds=contentType&displayedFieldIds=updatedAt&displayedFieldIds=author&page=0

### CODE REVIEW

File: apps/crn-server/test/integration/reminders.integration-test.ts [hunk 1]
```diff
@@ -200,8 +200,8 @@ describe('Reminders', () => {
 
       app = AppHelper(() => loggedInUser);
     });
-
-    describe('Published Reminder', () => {
+    // eslint-disable-next-line jest/no-disabled-tests
```
Comment (peterstarling) [inline apps/crn-server/test/integration/reminders.integration-test.ts:203]: maybe lets merge this
https://github.com/yldio/asap-hub/pull/3899
then rebase?

**Review by peterstarling — CHANGES_REQUESTED** [review]
please add tests

File: apps/crn-server/test/integration/reminders.integration-test.ts (hunk 1 above)
Comment (marianasrv) [reply apps/crn-server/test/integration/reminders.integration-test.ts:203]: yes sure

Comment (marianasrv) [reply apps/crn-server/test/integration/reminders.integration-test.ts:203]: this was so i could get it deployed, test it and see the logs

**Review by peterstarling — APPROVED** [review]

**Pull request merged**